examples/run_dpo.py

```diff
@@ -195,17 +200,49 @@ def setup_data(data_config: DataConfig, policy_config: PolicyConfig):
         max_seq_length=data_config["max_input_seq_length"],
     )
 
-    val_dataset = AllTaskProcessedDataset(
-        val_dataset,
-        tokenizer,
-        dpo_task_spec,
-        dpo_preprocessor,
-        max_seq_length=data_config["max_input_seq_length"],
-    )
+    val_dataset = {
```

Comment:
I think some typing must be updated because there are typing statements elsewhere like `val_dataset: AllTaskProcessedDataset`, but now it's a dict.